docs/src/constants/redirect-routes.js

```diff
@@ -138,6 +138,6 @@ export default {
   '/tokens/layout': ROUTES.SPACINGS,
   '/tokens/colors': ROUTES.COLORS,
   '/using/contributing': ROUTES.USING_BACKPACK,
-  '/style-guide/copywriting': ROUTES.COPYWRITING,
-  '/style-guide/tone-of-voice': ROUTES.TONE_OF_VOICE,
+  '/style-guide/copywriting': ROUTES.GUIDELINES,
+  '/style-guide/tone-of-voice': ROUTES.GUIDELINES,
```

Comment:
Shouldn't these be something like this?
```suggestion
  '/style-guide/copywriting': ROUTES.GENRERATED_GUIDELINES_COPYWRITING,
  '/style-guide/tone-of-voice': ROUTES.GENRERATED_GUIDELINES_TONE_OF_VOICE
```

Reply:
Yeah you're probably right. These redirect routes confuse me tbh, I kinda wanna delete most (or..all?) of them and just make broken links go to our home page. It feels kinda yagni to maintain all these links for years